rfcs/convergence/authoring-stories.md

```diff
@@ -261,17 +261,112 @@ export const parameters = { controls: { expanded: true } };
 
 ### 5. how to author e2e suites for those stories
 
-TBA
+> This section assumes that the storybook stories forms the core documentation of Fluent converged.
+> This may change in the future so the measures proposed in this section might no longer be relevant
+> if this assumption is no longer true.
 
-### 6. how to properly annotate stories with TS metadata to get the best DX possible
+`react-menu` already has entire test suites dedicated to testing against storybook. Since each PR build will
+deploy a storybook, the cypress testing solution for converged components will test against the PR storybook. This
+includes the constraint that until a component is added to `react-components` it cannot be tested in CI.
 
-TBA
+The e2e testing solution for `react-menu` can be used for any converged component using storybook, and is also used by
+`react-accordion` and `react-popover`. The purposes of E2E tests currently is to run internaction tests against the browser.
+Visual regression tests are handled by screener in `app-vrtests` and is out of the scope of this RFC.
+
+We propose to write an NX workspace generator to setup the e2e testing folder structure and dependencies for each package. The process is quite simple currently but still needs to be done manually.
```

Comment:
We can move to separate project, in another proposal since it changes more drastically the architecture of the repo. I'm just trying to build on what is already there.

Reply:
> it changes more drastically the architecture of the repo

thats a bit exaggerated don't you think ? :) from my POV its a straightforward change, but sure let's address this in separate RFC/proposal